web/problems/urls.py

```diff
@@ -20,7 +20,7 @@
         name="problem_edit_file",
     ),
     path(
-        "<int:problem_pk>/move/<int:shift>/",
+        "<int:problem_pk>/move/<shift>/",
```

Comment:
Tole je treba rešiti. Še bolje bi bilo, da se uporabi isto kot pri premikanju sklopov v predmetu.

Reply:
Se je javil @k3ap, da ima že skoraj spisano rešitev. Do zvečer še počakam s pull requestom, sicer pa ne bo nič narobe, če se to reši tudi kasneje.

Reply:
Trenutno nimam dovoljenja za push na ta branch; ali naj kar ustvarim nov PR iz k3ap/projekt-tomo na ul-fmf/projekt-tomo?